javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Weather app</title>
+    <script src="week1.js"></script>
+</head>
+<body>
+    <section class="top-banner">
+        <div class="container">
+          <h1 class="heading">HackYouFuture Weather App</h1>
+          <form>
+            <input type="text" placeholder="Search for a city" autofocus>
+            <button type="submit">search</button>
+            <span class="msg"></span>
+          </form>
+        </div>
+      </section>
+      <section class="ajax-section">
+        <div class="container">
+          <ul class="cities"></ul>
+        </div>
+      </section>
+      <footer class="page-footer">
+        <div class="container">
+          
+        </div>
+      </footer>
```

Comment:
here your code will look like this :`.... </footer><script src="week1.js"></script></body></html>`